granulate_utils/linux/ns.py

```diff
@@ -315,6 +314,8 @@ def resolve_host_path(process: Process, ns_path: str, from_ancestor: bool = True
 
 
 def get_host_pid(nspid: int, container_id: str) -> Optional[int]:
+    from granulate_utils.linux.containers import get_process_container_id
```

Comment:
in order to solve circular import:
`granulate_utils.linux.ns` imports `get_process_container_id` from `granulate_utils.linux.containers` which  imports `get_cgroups` from `granulate_utils.linux.cgroups`
and `granulate_utils.linux.cgroups` imports `resolve_host_root_links` from `granulate_utils.linux.ns`
@d3dave any suggestions?

Reply:
Instead of importing the name, you can import the module and then use `containers.get_process_container_id`.

Or alternatively, it makes sense to move `get_process_container_id` to `linux.process`

Reply:
>  it makes sense to move get_process_container_id to linux.process
prefer not to break the api

Reply:
> import the module and then use containers.get_process_container_id

doesnt solve it :/